rq_dashboard/templates/rq_dashboard/job.html

```diff
@@ -9,8 +9,10 @@ <h2><strong>Job ID</strong>: {{ id }}</h2>
 
         <span class="col-2">
             <button id="requeue-job-btn" class="btn btn-outline-warning btn-sm" style="display: none">Requeue</button>
-            {% if enable_delete %}
+            {% if enable_delete and registry_name != 'started' %}
             <button id="delete-job-btn" class="btn btn-outline-danger btn-sm">Delete</button>
+            {% elif enable_delete and registry_name == 'started' %}
+            <button id="stop-job-btn" class="btn btn-outline-danger btn-sm">Stop</button>
```

Comment:
### TODO

Here, we do not have `registry_name` and `d` as variables.
So we cannot get the registry name and job status to put `Delete` or `Stop` button.
This code puts `Delete` button everytime.

Reply:
done!